SDWebImage/FLAnimatedImage/FLAnimatedImageView+WebCache.m

```diff
@@ -47,30 +47,33 @@ - (void)sd_setImageWithURL:(nullable NSURL *)url
                   progress:(nullable SDWebImageDownloaderProgressBlock)progressBlock
                  completed:(nullable SDExternalCompletionBlock)completedBlock {
     __weak typeof(self)weakSelf = self;
+    dispatch_group_t group = dispatch_group_create();
     [self sd_internalSetImageWithURL:url
                     placeholderImage:placeholder
                              options:options
                         operationKey:nil
                        setImageBlock:^(UIImage *image, NSData *imageData) {
-                           // This setImageBlock may not called from main queue
                            SDImageFormat imageFormat = [NSData sd_imageFormatForImageData:imageData];
-                           FLAnimatedImage *animatedImage;
                            if (imageFormat == SDImageFormatGIF) {
-                               animatedImage = [FLAnimatedImage animatedImageWithGIFData:imageData];
+                               // Firstly set the static poster image to avoid flashing
+                               weakSelf.image = image;
```

Comment:
Well...Maybe we should just grab the first image if image is a `_UIAnimatedImage`.